sdcm/mgmt/cli.py

```diff
@@ -399,8 +400,8 @@ def wait_for_status(self, list_status, check_task_progress=True, timeout=3600, s
             return wait.wait_for(func=self.is_status_in_list, step=step, throw_exc=True,
                                  text=text, list_status=list_status, check_task_progress=check_task_progress,
                                  timeout=timeout)
-        except WaitForTimeoutError as ex:
-            raise WaitForTimeoutError(
+        except RetryError as ex:
```

Comment:
I would recommend waiting for both, it's not clear to me what's the change that broke it.

Reply:
`WaitForTimeoutError` does not exist on `manager-3.1`

Reply:
It is sequence of PRs that broke it.
Initially https://github.com/scylladb/scylla-cluster-tests/pull/6239 replaced `RetryError` with `WaitForTimeoutError`.
It was merged to the `master`, but wasn't to the `manager-3.1`

Next https://github.com/scylladb/scylla-cluster-tests/commit/444d9a43b70060821cc4e04f3caec7cfb84cba9a, was cherr-picked from `master` to `manager-3.1`, which is expected for `WaitForTimeoutError` to popup.

Reply:
So we can just backport https://github.com/scylladb/scylla-cluster-tests/pull/6239 ?

Reply:
Done, let's close this